sdk/communication/communication-sms/samples/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "typescript/dist",
+    "lib": ["DOM", "ES6"]
```

Comment:
We probably don't need to include DOM but fine if other service samples usually include it.